susePublicCloudInfoClient/Makefile

```diff
@@ -7,7 +7,9 @@ files = Makefile README.md LICENSE pint setup.py
 
 nv = $(shell rpm -q --specfile --qf '%{NAME}-%{VERSION}\n' *.spec)
 verSpec = $(shell rpm -q --specfile --qf '%{VERSION}' *.spec)
-verSrc = $(shell cat lib/susepubliccloudinfoclient/VERSION)
+verSrc = $(shell cat lib/susepubliccloudinfoclient/version.py | \
+	grep __version__ | awk -F\' '{ print $$2 }' | tr -d '\n')
+
```

Comment:
you could use python to read the version

```
$(shell python -c 'from susepubliccloudinfoclient/version import __version__; print __version__')
```

afaik the coding style also suggest big letters for private global variables __version__ vs. __VERSION__